src/planscape/planning/filters.py

```diff
@@ -0,0 +1,29 @@
+from django_filters import rest_framework as filters
+from planning.models import PlanningArea, Scenario, RegionChoices
+
+
+class PlanningAreaFilter(filters.FilterSet):
```

Comment:
Maybe we don't need this class, but keeping it here for convenience, in case we need to write custom filters for things like Region